torch_xla/core/xla_model.py

```diff
@@ -172,6 +176,9 @@ def get_ordinal(defval=0):
   Returns:
     The replication ordinal of the current process.
   """
+  if pjrt.using_pjrt():
```

Comment:
nit, should we update the docString to this function? I think the PJRT case we want to replication ordinal of the current thread? Not sure if here is the right place to explain the PJRT + v3 complication. same for the get_local_ordinal.

Reply:
Good catch, updated these docstrings.